src/DecisionTree.cpp

```diff
@@ -1835,7 +1835,7 @@ double DecisionTree::probabilityOfAClassGivenSequenceOfFeaturesAndValuesOrThresh
 bool DecisionTree::checkNamesUsed(const vector<string> &featuresAndValues)
 {
     for (const auto &featureAndValue : featuresAndValues) {
-        std::regex pattern(R"(\S+)\s*=\s*(\S+)");
+        std::regex pattern(R"((\S+)=([\S]+))");
```

Comment:
This change was required to match `feature=value` type strings given by evaluate, otherwise would throw regex error. all tests still pass.